pages/player/[uuid]/flips/[flipUid].tsx

```diff
@@ -21,7 +21,7 @@ function Flipper(props: Props) {
     let targetFlip = parseFlipTrackingFlip(props.targetFlip)
 
     function getTargetFlipEmbedDescription(targetFlip: FlipTrackingFlip) {
-        return `${targetFlip.profit > 0 ? '📈 Profit' : '📉 Loss'}:  ${numberWithThousandsSeperators(targetFlip.profit)} Coins
+        return `${targetFlip.profit > 0 ? '📈 Profit' : '📉 Loss'}:  ${numberWithThousandsSeperators(targetFlip.profit)} Coins ${targetFlip.profit > 0 ? `(${Math.round((targetFlip.profit / targetFlip.pricePaid) * 100)}%)` : ''}
```

Comment:
```suggestion
        return `${targetFlip.profit > 0 ? '📈 Profit' : '📉 Loss'}:  ${numberWithThousandsSeperators(targetFlip.profit)} Coins ${targetFlip.profit > 0 ? `(${Math.round((targetFlip.profit / targetFlip.pricePaid) * 98)}%)` : ''}
```
there is an ah tax of 1% to list and 1% on sell values over 1m